Adds a script that helps me run the debugger-extension unit tests
easier. Probably not needed if I understood the Ruby testing system
better.

diff --git a/test/debugger-ext/testit.sh b/test/debugger-ext/testit.sh
new file mode 100755
index 0000000..3f1a94e
--- /dev/null
+++ b/test/debugger-ext/testit.sh
@@ -0,0 +1,12 @@
+#!/bin/bash
+if (( $# == 0 )) ; then
+    args=test_*.rb
+else
+    args=$*
+fi
+for t in $args ; do 
+    echo "===== $t ====="
+    ../../miniruby -I../../lib -I../.. $t
+    # echo ../../miniruby -I../../lib -I../.. $t
+    # gdb ../../miniruby
+done
